modules/ting_search_ortograf/includes/ting_search_ortograf.admin.inc

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * @file
+ * Admin settings for bibliotek.dk autocomplete.
+ */
+
+/**
+ * Ting Search Ortograf admin form
+ */
+function ting_search_ortograf_admin() {
+  $default_settings = ting_search_ortograf_settings();
+
+  // Attach the CSS to the form.
+  $path = drupal_get_path('module', 'ting_search_ortograf');
+  $form['#attached'] = array(
+    'css' => array(
+      'type' => 'file',
+      'data' => $path . '/css/ting_search_ortograf_admin_form.css',
+    ),
+  );
+
+  $form['ting_search_ortograf_webservice'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Autocomplete URL', array(), array('context' => 'ting_search_ortograf')),
+    '#tree' => FALSE,
+    '#collapsible' => TRUE,
+  ); 
+
+  $form['ting_search_ortograf_webservice']['ting_search_ortograf_suggestion_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Autocomplete URL', array(), array('context' => 'ting_search_ortograf')),
+    '#size' => 73,
+    '#description' => t('URL to Autocomplete webservice. F.ex.: https://ortograf.dbc.dk/ortograf/suggest', array(), array('context' => 'ting_search_ortograf')),
+    '#default_value' => variable_get('ting_search_ortograf_suggestion_url', NULL),
+    '#required' => TRUE,
+  );
+
+  $form['ting_search_ortograf_settings'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Autocomplete settings', array(), array('context' => 'ting_search_ortograf')),
+    '#tree' => TRUE,
+    '#collapsible' => TRUE,
+  );
+
+  $form['ting_search_ortograf_settings']['rows'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Max suggestions', array(), array('context' => 'ting_search_ortograf')),
+    '#size' => 26,
+    '#default_value' => $default_settings['rows'],
+  );
+
+  $form['ting_search_ortograf_settings']['minimumString'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Minimum string', array(), array('context' => 'ting_search_ortograf')),
+    '#description' => t('Minimum string length to do autocomplete lookup on.', array(), array('context' => 'ting_search_ortograf')),
+    '#size' => 26,
+    '#default_value' => $default_settings['minimumString'],
```

Comment:
Validation like [`element_validate_integer_positive`](https://api.drupal.org/api/drupal/includes%21form.inc/function/element_validate_integer_positive/7.x) would be nice here.